h/api_client/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+"""A Python client for the Hypothesis API."""
+from h.api_client.api_client import Client, APIError, ConnectionError, Timeout
```

Comment:
I've been starting to move toward using relative imports where possible in our codebase. How do you feel about changing this to `from .api_client import ...`.